Lib/test/bisect_cmd.py

```diff
@@ -51,6 +51,12 @@ def python_cmd():
     cmd = [sys.executable]
     cmd.extend(subprocess._args_from_interpreter_flags())
     cmd.extend(subprocess._optim_args_from_interpreter_flags())
+    for key, value in sys._xoptions.items():
+        cmd.append("-X")
+        if value == True:
```

Comment:
```suggestion
        if value is True:
```

Reply:
Eh, why? It's a bunch of strings that are probably (definitely?) not going to override `__eq__`

Reply:
See second point here: https://peps.python.org/pep-0008/#programming-recommendations

Reply:
Huh, TIL. Thanks